src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -213,13 +213,23 @@ case class MesosStatusUpdateEvent(
 
 case class InstanceChanged(
     id: Instance.Id,
+    runSpecVersion: Timestamp,
```

Comment:
I think we should ultimately move timestamp over to ODT. I can do this separately.